tests/conftest.py

```diff
@@ -13,11 +13,11 @@ def pytest_sessionstart(session):
     # this is for local unit testing with google appengine, otherwise you get a
     # 'No api proxy found for service "urlfetch"' response
     try:
-        from google.appengine.ext import testbed
+        from google.appengine.ext import testbed  # type: ignore
 
-        session.appengine_testbed = testbed.Testbed()
-        session.appengine_testbed.activate()
-        session.appengine_testbed.init_urlfetch_stub()
+        session.appengine_testbed = testbed.Testbed()  # type: ignore
+        session.appengine_testbed.activate()  # type: ignore
+        session.appengine_testbed.init_urlfetch_stub()  # type: ignore
```

Comment:
The `#type: ignore` is an indicator to `mypy` to ignore type-hinting issues here. Otherwise, this line is a nightmare because it could realistically be several different types and that spreads up the chain to the rest of the app.